rpc/client/interface.go

```diff
@@ -24,17 +24,27 @@ import (
 	"context"
 
 	"github.com/tendermint/tendermint/libs/bytes"
-	"github.com/tendermint/tendermint/libs/service"
 	ctypes "github.com/tendermint/tendermint/rpc/core/types"
 	"github.com/tendermint/tendermint/types"
 )
 
 //go:generate ../../scripts/mockery_generate.sh Client
 
-// Client wraps most important rpc calls a client would make if you want to
-// listen for events, test if it also implements events.EventSwitch.
+// Client is the interface that must be satisfied by implementations of a
+// client of the Tendermint RPC service.
```

Comment:
this is sexy!

Reply:
I would say something like: 

"Client describes the interface of Tendermint RPC clients implementations" (e.g. "client is the interface that must be satisfied by implementations of a client" uses many words to say very little.)